#--- etc/Makefile.orig	2011-03-17 02:56:21.000000000 +0300
#+++ etc/Makefile	2011-04-29 21:32:07.000000000 +0400
#@@ -22,8 +22,8 @@
# 
# .PHONY: install
# install:
#-	install -d -m 755 /etc
#-	install -d -m 755 /etc/init.d
#-	install -m 700 $(RCFILE) /etc/init.d/
#+	install -d -m 755 $(DESTDIR)/etc
#+	install -d -m 755 $(DESTDIR)/etc/rc.d
#+	install -m 700 $(RCFILE) $(DESTDIR)/etc/rc.d/
# 	test ! -x /sbin/insserv || /sbin/insserv $(RCFILE)
# 
--- kernel/Makefile.orig	2011-04-29 22:33:44.000000000 +0400
+++ kernel/Makefile	2011-04-29 22:34:18.000000000 +0400
@@ -51,8 +51,7 @@
 	rm -rf .tmp_versions
 
 install:
-	install -o root -g root -m 644 mhvtl.ko /lib/modules/`uname -r`/kernel/drivers/scsi/; \
-	depmod -a; \
+	install -D -o root -g root -m 644 mhvtl.ko $(DESTDIR)/lib/modules/`uname -r`/kernel/drivers/scsi/mhvtl.ko; \
 	if [ -f "/etc/modprobe.d/unsupported-modules" ]; then \
 		sed -e 's/^allow_unsupported_modules 0/allow_unsupported_modules 1/' /etc/modprobe.d/unsupported-modules > /etc/new; \
 		/bin/mv /etc/new /etc/modprobe.d/unsupported-modules; \
